UTAT_Compression_Algorithm/C/helperlib.c

```diff
@@ -0,0 +1,51 @@
+#include <math.h>
```

Comment:
Could you either move the includes to the top level file (compression.h) or use include guards? 

ex. #ifndef GRANDPARENT_H
#define GRANDPARENT_H

struct foo {
    int member;
};

#endif /* GRANDPARENT_H */